docs/about/compatibility/precission-support.rst

```diff
@@ -0,0 +1,544 @@
+    *
+      - float16
+      - ``half``
+      - A 16-bit floating-point number, conforming to the IEEE 754-2008 half precision storage format.
+    *
+      - bfloat16
+      - ``bfloat16``
+      - A shortened 16-bit version of the IEEE 754 single precision storage format.
+    *
+      - tensorfloat32
+      - ``-``
+      - Floating-point number that occupies 32 bits or less of storage, providing improved range compared to half (16-bit) formats, at (potentially) greater throughput than single precision (32-bit) formats.
+    *
+      - float32
+      - ``float``
+      - A 32-bit floating-point number, conforming to the IEEE 754 single precision storage format.
+    *
+      - float64
+      - ``double``
+      - A 64-bit floating-point number, conforming to the IEEE 754 double precision storage format.
+
+.. note::
+
+  * The float8 and tensorfloat32 types are internal types used in calculations in the Matrix Cores and can be stored in any type of the same size.
+  * The encodings for FP8 (E5M2) and FP8 (E4M3) natively supported by MI300 differ from the FP8 (E5M2) and FP8 (E4M3) encodings used in H100 ([FP8 FORMAT SFOR DEEP LEARNING](https://arxiv.org/abs/2209.05433)).
+  * In some AMD documents and articles float8 (E5M2) is called bfloat8.
+
+The support icon description
+==========================================
+
+.. list-table::
+    :header-rows: 1
+
+    * - The support icon explanation
+      - Not supported
+      - Partial support
+      - Support without hardware acceleration
+      - Full support
+
+    * - Support icon
+      - ❌
+      - ⚠️
+      - ✔
+      - ✅
+
+.. note::
+
+  Support without hardware acceleration means, that these types are emulated,
+  and the users can get close to the theoretical limits of the operations per
+  seconds. Example 16 bit integers operation calculated on 32 bit integer units
+  and reach the same throughput as int32 operations.
+
+Native hardware support
+==========================================
+
+Hardware accelerated compute units support
+-------------------------------------------------------------------------------
+
+Integer types
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. tab-set::
+
+  .. tab-item:: Integer types
+    :sync: integer-type
+
+    .. list-table::
+      :header-rows: 1
+
+      * 
+        - Integer types name
+        - int8
+        - int16
+        - int32
+        - int64
+      * 
+        - Mi100
+        - ✅
+        - ✅
+        - ✅
+        - ✅
+      * 
+        - Mi200 series
+        - ✅
+        - ✅
+        - ✅
+        - ✅
+      * 
+        - Mi300 series
+        - ✅
+        - ✅
+        - ✅
+        - ✅
+
+  .. tab-item:: Floating-points types
+    :sync: floating-point-type
+
+    .. list-table::
+      :header-rows: 1
+
+      * 
+        - Floating-point types name
+        - float8 (E4M3)
+        - float8 (E5M2)
```

Comment:
Not really important, but something that annoys me: Can the names of the float8 types be force to be rendered on one line? Then the site wouldn't jump around, when switching from integral types to floating-point types.

Reply:
We do not have enough space for that.